graph.go

```diff
@@ -28,6 +36,14 @@ func NewGraph(title, tmpl string) Graph {
 		ScvgSys:      []graphPoints{},
 		ScvgReleased: []graphPoints{},
 		ScvgConsumed: []graphPoints{},
+		STWSclock:    []graphPoints{},
+		MASclock:     []graphPoints{},
+		STWMclock:    []graphPoints{},
+		STWScpu:      []graphPoints{},
+		MASAssistcpu: []graphPoints{},
+		MASBGcpu:     []graphPoints{},
+		MASIdlecpu:   []graphPoints{},
+		STWMcpu:      []graphPoints{},
```

Comment:
> without these fields set, they don't get included in json and they don't show up on the graphs.

urgh, I remember now that was the reason, thanks

>  they will show up as soon as there's data for them but i think it's a good practice to show all fields even before the data is there, it's just a little clearer I find.

I'll have to disagree with you there, but that's not important.